docs/_includes/sidebar.html

```diff
@@ -41,15 +41,15 @@
   </ul>
 
   <ul class="sidebar-section">
-    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#steps">
-      <li class="title">{{ site.t[page.lang].steps }}</li>
-    </a>
-    {% assign workflow_steps = site.steps | sort: "order" | where: "lang", page.lang %} {% for step in workflow_steps %}
-      {% if step.slug != "steps-overview" %}
-      <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#{{ step.slug }}">
-        <li>{{step.title}}</li>
+    <li class="title">
+      Building Run on Slack apps
```

Comment:
I'm thinking we should call this something else since bolt apps don't run on slack

Reply:
`Adding next generation features` or `Adding functions, workflows and triggers`?